### PR TITLE
feat(hapi): upgrade to hapi 18

### PR DESCRIPTION
**Pull request by pi0**
- Upgrade to `@hapi/hapi` v18 and add explicitly dependency
- Use `@nuxtjs/hapi` instead of `hapi-nuxt`
- Improve startup script to use `async/await` + global rejection handler

### CODE REVIEW

File: template/frameworks/hapi/server/index.js [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
 const consola = require('consola')
-const HapiNuxt = require('hapi-nuxt')
+const Hapi = require('@hapi/hapi')
+const HapiNuxt = require('@nuxtjs/hapi')
 
-const server = new Hapi.Server({
-  host: process.env.HOST || '127.0.0.1',
-  port: process.env.PORT || 3000
-})
+function start() {
```
Comment (halimb) [inline template/frameworks/hapi/server/index.js]: @clarkdo Shouldn't the function be `async`? (I'm having an error at this line on fresh installs)

Comment (pi0) [reply template/frameworks/hapi/server/index.js]: Nice catch! Will add a fix.

Comment (pi0) [reply template/frameworks/hapi/server/index.js]: Already [fixed](https://github.com/nuxt/create-nuxt-app/commit/a8ecdb93476753a2e2d0ab73504c73c03c985d1a#diff-ab6abf3f2a5af12e7f8b2e276a5c9ea2R5). waiting for release.

Comment (clarkdo) [reply template/frameworks/hapi/server/index.js]: @halimb @pi0 v2.8.0 has been released.

**Pull request merged**